client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -552,6 +587,78 @@ export const confirmPackages = ( orderId, siteId ) => ( dispatch, getState ) =>
 	tryGetLabelRates( orderId, siteId, dispatch, getState );
 };
 
+export const setContentsType = ( orderId, siteId, packageId, value ) => ( {
+	type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SET_CONTENTS_TYPE,
+	siteId,
+	orderId,
+	packageId,
+	value,
```

Comment:
Would be cool to name the value what it's supposed to represent, something like `contentsType` in this case, for easier reading, & scanning.

Reply:
Changed for all the actions in https://github.com/Automattic/wp-calypso/pull/24974/commits/056b152215aa36d2f83fc22c6c8402bc694875ee